blocks/header/header.css

```diff
@@ -0,0 +1,187 @@
+.header {
+  padding: 10px;
+  position: fixed;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.header_scrolled {
+  padding: 0;
+}
+
+.header_open {
+  height: 100vh;
+}
+
+.header__content {
+  background-color: transparent;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  height: 36px;
+  overflow: hidden;
+  transition: height 0.3s ease, background-color 0.3s ease;
+}
+
+.header__content_scrolled {
+  background-color: #683d83;
+  box-shadow: 0 0px 12px rgb(0 0 0 / 30%);
+}
+
+.header__content_open {
+  height: 216px;
+  background-color: #fff;
+  box-shadow: 0 0px 12px rgb(0 0 0 / 30%);
+}
+
+.header__logo-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 44px;
+}
+
+.header__logo {
+  width: 100px;
+  fill: #fff;
+  transition: fill 0.2s ease, opacity 0.2s ease;
+  opacity: 1;
+  cursor: pointer;
+}
+
+.header__logo:hover {
+  fill: #683d83;
+}
+
+.header__logo_scrolled:hover {
+  /* fill: #fff; */
+  fill: #fff;
+  opacity: 0.7;
+}
+
+.header__logo_opened {
+  fill: #000;
+}
+
+.header__logo_opened:hover {
+  fill: #000;
+}
+
+.header__logo:hover .header__logo-accent {
+  fill: #a791b8;
+}
+
+.header__logo-accent {
+  fill: #a6a6a6;
+  transition: fill 0.2s ease;
+}
+
+.header__hamburger {
+  width: 30px;
+  cursor: pointer;
+}
+
+.header__links {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.header__link {
+  text-transform: uppercase;
+  font-weight: 700;
+  text-decoration: none;
+  color: #000;
+  font-size: 20px;
+  line-height: 24px;
+  margin: 0;
+  margin-bottom: 20px;
+  transition: color 0.1s ease;
+}
+
+.header__link:hover {
+  color: #683d83;
+}
+
+@media (min-width: 768px) {
```

Comment:
Feel this is a bit big for the first query as hamburger should really just be for mobile sizes. 480px is generally a good breakpoint for a mobile-only style.

Reply:
Okay I had some text-wrapping troubles when I just tried to change the breakpoint (you're right 768px is huuuge) and had an idea for a fun _feature_... it might suck. Let me know what you think. I shot a little video of a mouseover video.

[Explanation](https://www.loom.com/share/36b79df4bae3449a8e6c0c15e87cb917)
Demo attached ⤵️

https://user-images.githubusercontent.com/74033573/121932258-acf86f00-cd12-11eb-9c64-f0c9c971269d.mov

Reply:
On click would probably be better than mouse-over for this. I'm imagining as a user how I might be looking at a different section and having the menu pop out because I moved my mouse too much would get annoying when I wasn't trying to interact with it.